packages/loader/container-loader/src/containerContext.ts

```diff
@@ -86,7 +86,20 @@ export class ContainerContext implements IContainerContext {
         return context;
     }
 
-    public readonly logger: ITelemetryLogger;
+    public readonly taggedLogger: ITelemetryLogger;
+
+    /**
+     * Subtlety: returns this.taggedLogger since vanilla this.logger is now deprecated. See IContainerContext for more
+     * details.
+    */
+    /** @deprecated See IContainerContext for more details. */
+    public get logger(): ITelemetryLogger {
+        return this.taggedLogger;
+    }
+    /** @deprecated See IContainerContext for more details. */
+    public set logger(telemetryLogger: ITelemetryLogger) {
+        this.logger = telemetryLogger;
```

Comment:
I think this is an infinite loop 😎

Reply:
You can just no-op in here I think...?  Or can we not have a setter since it was readonly?